### ISSUE TITLE
[cetic/nifi] Custom configmap support

### ISSUE THREAD

**Issue by novakov-alexey** [cetic/helm-nifi#48]
**Is your feature request related to a problem? Please describe.**
I need to mount my own CofigMaps to NiFi Pods. Current Charts do not have such feature, however, it is similar to custom Secrets, which is supported already

**Describe the solution you'd like**
I would like to mount my custom ConfigMaps via Helm values and `--set` option.

**Describe alternatives you've considered**
There is no clear alternative rather than fork this repo and add custom ConfigMaps into Pod spec of the NiFi Statefulset

**Additional context**
I can propose a PR, which will allow to add ConfigMaps in values similar to Secrets:
```
configmaps:
  - name: nifi-krb5-conf
    keys:
      - krb5.conf
    mountPath: /etc/krb5.conf
```
Would you be interested in such PR?

**Comment by novakov-alexey** [comment]
PR has been created

**Comment by alexnuttinck** [comment]
Hello @novakov-alexey,
Thanks, I review your PR, it will be useful!

**Comment by alexnuttinck** [comment]
Merged in [0.3.3](https://github.com/cetic/helm-nifi/releases/tag/0.3.3). Thanks @novakov-alexey
